blockchain/reactor.go

```diff
@@ -134,6 +132,8 @@ func (bcR *BlockchainReactor) Receive(chID byte, src *p2p.Peer, msgBytes []byte)
 
 	bcR.Logger.Debug("Receive", "src", src, "chID", chID, "msg", msg)
 
+	// [zr] note: the `megacheck` tool (go vet on steroids)
+	// really dislikes the logic in this switch
```

Comment:
this is also the reason I wouldn't add `megacheck` to Makefile _yet_, because some (e.g., this one) code blocks were left as-is since I don't know enough about the codebase to refactor-to-not-fail-the-megacheck